data/sql/updates/pending_db_world/dartols-rod.sql

```diff
@@ -0,0 +1,11 @@
+DELETE FROM `gossip_menu` WHERE `MenuID` = 5501;
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`) VALUES (5501, 6554);
```

Comment:
insert without delete, we should always make sure to delete so that the query doesnt get stuck!

Reply:
Unless my Github is bugging out, there is already a delete for the insert.

Reply:
no my eyes betrayed me <3